app/features-json/graph_ql/queries/project_query.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "../types/project_type"
+
+module FastlaneCI
+  module Query
+    Projects = GraphQL::ObjectType.define do
+      name "Projects"
+      field :projects, types[Types::ProjectType] do
+        argument(:id, types.String)
+        description "Get a list of Projects"
+        resolve lambda do |_obj, args, _ctx|
+          if !args[:id].nil?
```

Comment:
Could we do an `unless` here instead please?

Reply:
I forgot about that - my bad 😳

Reply:
could we reverse the order of this condition?

```
if nil -> all projects
else -> get project
```

Reply:
Because then you're skipping all the other parameters that might be present in the query itself (that are not added for now), like, projects with certain name regex, or something else.